code/game/objects/items/weapons/material/swords.dm

```diff
@@ -70,7 +70,7 @@
 	
 /obj/item/weapon/material/sword/axe
 	name = "battle axe"
-	desc = "A one handed battle axe, still a deadly weapon."
+	desc = "An one handed battle axe, still a deadly weapon."

```

Comment:
Actually. Hate to be a downer, but it's still "A one handed battle axe." The article is dependent on the noun it's associated with (battle axe), not the adjectives (one handed) that come before it.